b3d/io/video_input.py

```diff
@@ -0,0 +1,134 @@
+from b3d.types import Array
```

Comment:
@mirkoklukas we have this already in https://github.com/probcomp/b3d/blob/2b0a264d7c845848ad1cf5c43e46ecb1fb834dde/b3d/utils.py#L491

Reply:
oh never mind. i see you remove that duplication